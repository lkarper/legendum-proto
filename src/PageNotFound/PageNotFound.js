import React from 'react';

const PageNotFound = (props) => {
    return (
        <section className='section'>
            <header>
                <h2>Page not found</h2>
            </header>
            <p>We couldn't find the page you're looking for.  Check the address and try again.</p>
        </section>
    );
}

export default PageNotFound;
