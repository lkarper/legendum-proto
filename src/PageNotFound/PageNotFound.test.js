import React from 'react';
import ReactDOM from 'react-dom';
import { mount } from 'enzyme';
import toJSON from 'enzyme-to-json';
import PageNotFound from './PageNotFound';

describe('PageNotFound component', () => {
    it('renders the UI without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<PageNotFound />, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders the UI as expected', () => {
        const wrapper = mount(<PageNotFound />);
        expect(toJSON(wrapper)).toMatchSnapshot();
        wrapper.unmount();
    });
});
