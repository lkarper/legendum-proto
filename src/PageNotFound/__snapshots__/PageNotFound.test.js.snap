// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound component renders the UI as expected 1`] = `
<PageNotFound>
  <section
    className="section"
  >
    <header>
      <h2>
        Page not found
      </h2>
    </header>
    <p>
      We couldn't find the page you're looking for.  Check the address and try again.
    </p>
  </section>
</PageNotFound>
`;
